README.md

```diff
@@ -19,7 +18,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Install
 - Set up Github Actions
-- Set up linter > Lighthouse , Webhint , Stylelint and Eslint.
+- Set up linter > Lighthouse, Webhint, Stylelint and Eslint.
 
 ### Run tests
 Open your favourite code editor (Optional VSCode), and type on the terminal the `npm run dev` command to see the results. 
```

Comment:
**[Optional]**
- [ ] Please add all the commands from the **Clone and Review it** section of this [readme](https://github.com/HAMMAS-SALEEM/ToDoList/blob/main/README.md#clone-and-review-it) file to your readme file. :blush: to make other people know how to set up your project :+1:

Reply:
Hello @HAMMAS-SALEEM,
Thank you so much for your suggestion 🙌🏻. I will apply it all to my future projects 😀. Have a nice week!